CI/values.yaml

```diff
@@ -41,6 +41,40 @@ service:
 ingress:
   subdomain: grid-tariff-api
   path: "/"
+ 
+ # Syncing of data from bigquery to database  can make the startup take a long time
+startupProbe:
+  failureThreshold: 125
+  httpGet:
+    path: /ping
+    port: 8080
+    scheme: HTTP
+  initialDelaySeconds: 5
+  periodSeconds: 5
+  successThreshold: 1
+  timeoutSeconds: 3
+
+readinessProbe:
+  failureThreshold: 
```

Comment:
Her mangler verdi for failureThreshold. Foreslår 5, så det blir likt som livenessProbe (de kan være forskjellige, men da må man passe på at de blir riktige i forhold til hverandre).